exampleSite/utilities/examples/content/events/yyyy-city/index.md

```diff
@@ -0,0 +1,86 @@
++++
+Title = "Welcome"
+Type = "welcome"
+aliases = ["/events/yyyy-city/welcome"]
++++
+
+<!-- <div style="text-align:center;">
+  {{< event_logo >}}
+</div> -->
+
+<div class = "row">
+  <div class = "col-md-2">
```

Comment:
I just left what existed in terms of that content - definitely go ahead and push changes to this branch if there's something you want to tweak?